File: year2/mpp/java/MotorcycleContest/src/main/java/domain/dtos/RaceInfo.java
package domain.dtos;

import domain.entities.Race;

public class RaceInfo {
    private Race race;
    private Integer participantCount;

    public RaceInfo(Race race, Integer participantCount) {
        this.race = race;
        this.participantCount = participantCount;
    }

    public Race getRace() {
        return race;
    }

    public void setRace(Race race) {
        this.race = race;
    }

    public Integer getParticipantCount() {
        return participantCount;
    }

    public void setParticipantCount(Integer participantCount) {
        this.participantCount = participantCount;
    }

    @Override
    public String toString() {
        return race.getId() + ". Capacity " + race.getCapacity() + "cc"+ ", Participants: " + participantCount.toString();
    }
}

File: year2/mpp/c#/MotorcycleContest/MotorcycleContest/Domain/Dtos/RaceInfo.cs
using System;

using MotorcycleContest.Domain.Entities;

namespace MotorcycleContest.Domain.Dtos
{
    public class RaceInfo
    {
        public Race Race { get; set; }
        public Double ParticipantCount { get; set; }

        public RaceInfo(Race race, Double participantCount)
        {
            this.Race = race;
            this.ParticipantCount = participantCount;
        }

        public override string ToString()
        {
            return "Race: " + Race.ID + ", Participants: " + ParticipantCount;
        }
    }
}
